pyhyp/pyHyp.py

```diff
@@ -778,6 +952,12 @@ def __del__(self):
         """
         self.hyp.releasememory()
 
+    def get_used_march_distance(self):
+        if self.gridGenerated:
+            return self.hyp.hypinput.marchdist
+        else:
+            raise Error("Can not returning used marching distance before extruding the grid")
```

Comment:
No need for the `else`, just check the inverse i.e., `if not self.gridGenerated` and throw the error, otherwise 
just `return` the member variable.
Also please rephrase the error string.